datamover/pkg/drivers/https/migration.go

```diff
@@ -610,7 +620,7 @@ func runjob(in *pb.RunJobRequest) error {
 					//update database
 					j.PassedCount = (int64(passedCount))
 					j.PassedCapacity = capacity
-					j.Progress = int64(capacity * 100 / j.TotalCapacity)
+					//j.Progress = int64(capacity * 100 / j.TotalCapacity)
 					logger.Printf("capacity:%d,TotalCapacity:%d Progress:%d\n", capacity, j.TotalCapacity, j.Progress)
```

Comment:
Progress will now be calculated separately. Don't we need to correct the logger statement too here?